src/Components/PetList.js

```diff
@@ -1,7 +1,26 @@
+import { useState, useEffect } from "react";
+
 export const PetList = () => {
+
+  const [pets, setPets] = useState([]);
+
+
+    useEffect(() => {
+    fetch(`https://vet-app-0obi.onrender.com/api/pets`)
```

Comment:
Your code works with the exception that this is the wrong url. You should use the one that includes `pets?employeeId=${id}`